components/dropdown/dropdown-context-menu.js

```diff
@@ -34,7 +34,7 @@ class DropdownContextMenu extends DropdownOpenerMixin(VisibleOnAncestorMixin(Lit
 
 	render() {
 		return html`
-			<d2l-button-icon aria-label=${this.text} ?disabled=${this.disabled} icon="tier1:chevron-down" text=${this.text} ?translucent=${this.translucent}>
+			<d2l-button-icon ?disabled=${this.disabled} icon="tier1:chevron-down" text=${this.text} ?translucent=${this.translucent}>
```

Comment:
FYI @AllanKerr I removed the `aria-label` here and in `dropdown-more` because `text` on `button-icon` also adds the `aria-label` to the `button`. Let me know if there were additional reasons why this `aria-label` was added.